src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -346,6 +405,179 @@ export class PointerTool extends BaseTool {
       };
     });
   }
+
+  async handleDragSelectionBoundsResize(selection, eventStream, initialEvent) {
+    const sceneController = this.sceneController;
+    const clickedResizeHandle = sceneController.sceneModel.clickedResizeHandle;
+
+    // The following may seem wrong to you, but it's correct.
+    // Because we say for example bottom-left and not left-bottom. Y-X order.
+    const resizeHandlePositionX = clickedResizeHandle.split("-")[1];
+    const resizeHandlePositionY = clickedResizeHandle.split("-")[0];
```

Comment:
You are repeating yourself too much: you call `.split("-")` twice. Easy fix:
```js
    const [resizeHandlePositionY, resizeHandlePositionX] = clickedResizeHandle.split("-");
```

Reply:
Thanks. done.